test/lit.cfg

```diff
@@ -1331,7 +1334,7 @@ def source_compiler_rt_libs(path):
                                         and config.compiler_rt_platform in lib])
 
 compiler_rt_dir = make_path(test_resource_dir, 'clang', 'lib',
-                            platform.system().lower())
+                            platform.system().lower() if not kIsAndroid else 'android')
```

Comment:
Several sanitizer tests' shared libraries in the compiler-rt build couldn't be found if this wasn't properly set.